README.md

```diff
@@ -18,3 +20,9 @@ transactional system.
 See the [intro](https://github.com/ubuntu-core/snapcraft/blob/master/docs/intro.md)
 for details about the concepts behind
 snapcraft. There is also a [tutorial](https://github.com/ubuntu-core/snapcraft/blob/master/docs/intro.md). If you're interested in developing Snapcraft you should look at the [hacking guide](https://github.com/ubuntu-core/snapcraft/blob/master/HACKING.md).
+
+[travis-image]: https://travis-ci.org/ubuntu-core/snapcraft.svg?branch=master
+[travis-url]: https://travis-ci.org/ubuntu-core/snapcraft.svg?branch=master
```

Comment:
Ah, wait, why don't you make the urls point to https://travis-ci.org/ubuntu-core/snapcraft instead of the svg?